picard/ui/options/general.py

```diff
@@ -74,21 +73,23 @@ def load(self):
         self.ui.server_port.setValue(config.setting["server_port"])
         self.ui.analyze_new_files.setChecked(config.setting["analyze_new_files"])
         self.ui.ignore_file_mbids.setChecked(config.setting["ignore_file_mbids"])
-        self.ui.check_for_updates.setChecked(config.setting["check_for_updates"])
-        self.ui.update_level.clear()
-        for level, description in PROGRAM_UPDATE_LEVELS.items():
-            self.ui.update_level.addItem(_(description['title']), level)
-        self.ui.update_level.setCurrentIndex(self.ui.update_level.findData(config.setting["update_level"]))
-        self.ui.update_check_days.setValue(config.setting["update_check_days"])
+        if QtWidgets.QApplication.instance().autoupdate_enabled:
```

Comment:
I believe that the value is exposed as `self.tagger.autoupdate_enabled` so the `instance()` object shouldn't be required.  Same throughout the file.